src/components/BuildPages/BuildPages.tsx

```diff
@@ -127,14 +129,19 @@ export const BuildPages = () => {
 
   const isLogged = !serviceContainerBuild.data?.status || isBuildWithLog(serviceContainerBuild.data.status);
   const isLoggedTooltip = !isLogged ? `Builds with status ${serviceContainerBuild.data!.status} are not logged.` : '';
+  const liveLogLinkTooltip = !isBuilding ? `Not available while not building.` : '';
```

Comment:
Here we could use some equivalent phrasing, something like `Build is not in progress` to avoid double `not`.